client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -380,7 +380,15 @@ const EditPage = createClass({
 
 **[Homebrewery Link](${global.config.publicUrl}/share/${shareLink})**`;
 
-		return `https://www.reddit.com/r/UnearthedArcana/submit?title=${encodeURIComponent(title)}&text=${encodeURIComponent(text)}`;
+		let encodedTitle;
+		try {
+			encodedTitle = encodeURIComponent(title);
+		} catch (error) {
+			console.error("Error encoding title for Reddit link:", error);
+			encodedTitle = encodeURIComponent("Check out my homebrew!"); // Fallback title
+		}
+	
+		return `https://www.reddit.com/r/UnearthedArcana/submit?title=${encodedTitle}&text=${encodeURIComponent(text)}`;
```

Comment:
Looks like `toWellFormed()` exists, which can take a string and replace any invalid characters with a placeholder. Might be more direct to fix the string so the error doesn't happen in the first place.

```suggestion
return `https://www.reddit.com/r/UnearthedArcana/submit?title=${encodeURIComponent(title.toWellFormed())}&text=${encodeURIComponent(text)}`;
```

Reply:
is this function used elsewhere in our code?

Reply:
I tested this with your broken document and can confirm `toWellFormed()` fixes the issue, so I will replace with that.

There is only one other place where we URI encode user-made content, when generating the filename for download. The download function already runs through the `sanitize-filename` package, which I tested and does handle this malformed title, so we don't need to change anything there.